db/seeds.rb

```diff
@@ -1,7 +1,21 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+def seed
+  unless Rails.env.development? || ENV["RUN_SEEDS_IN_PRODUCTION"]
```

Comment:
This seems dangerous and currently unneeded?

Reply:
Yes, I copied it from https://github.com/alphagov/specialist-publisher/blob/master/db/seeds.rb#L2-L5 without really questioning

Fixed in c65785d5e8d64c5a2a5b4cc39ab9ce5e5bf81c1e